cmd/kubernetes_create.go

```diff
@@ -15,15 +15,17 @@ import (
 )
 
 var numTargetNodes int
-var waitKubernetes, saveConfigKubernetes, mergeConfigKubernetes, switchConfigKubernetes bool
-var kubernetesVersion, targetNodesSize, clusterName, applications, removeapplications, networkID, existingFirewall, createFirewall, cniPlugin string
+var waitKubernetes, saveConfigKubernetes, mergeConfigKubernetes, switchConfigKubernetes, createFirewall bool
+var kubernetesVersion, targetNodesSize, clusterName, applications, removeapplications, networkID, existingFirewall, cniPlugin, rulesFirewall string
 var kubernetesCluster *civogo.KubernetesCluster
 
 var kubernetesCreateCmdExample = `civo kubernetes create CLUSTER_NAME [flags]
 
 Notes:
+* The '--create-firewall' It will be create the default rules for the firewall.
```

Comment:
Above it says that creating a firewall will open all ports, and here "default" rules? Are they the same thing?

Grammatically, I suggest

`* The '--create-firewall' will add add default rules for the firewall if none are specified.`

That's assuming the "default rules" are defined somewhere.